composer.json

```diff
@@ -13,7 +13,8 @@
         "php": "^7.1",
 
         "behat/behat": "^3.1",
-        "symfony/http-kernel": "^3.3|^4.0"
+        "symfony/http-kernel": "^3.3|^4.0",
+        "symfony/dotenv": "^4.0"
```

Comment:
Can we go for `^3.3|^4.0`?

Reply:
Let's also add a line similar to https://github.com/FriendsOfBehat/SymfonyExtension/blob/db6193144e1f738a2e5ec201d1937c94db95259d/.travis.yml#L20 in `.travis.yml` but for this dependency.